### PR TITLE
INTLY-7439 Verify 3scale SMTP configuration

### PR DESCRIPTION
**Pull request by CathalOConnorRH**
# Description
Automated test to confirm 3scale can send an email 

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by philbrookes** [comment]
This looks good, but I'd like it to be rebased on that PR for not blocking on SMTP and retest on top of that. Once that has merged. WDYT?

File: test/common/3scale_smtp.go [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app": "smtp-server",
+					},
+				},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
+						{
+							Name:  "smtp-server",
+							Image: "quay.io/cathaloconnor/smtp-server:latest",
```
Comment (briangallagher) [inline test/common/3scale_smtp.go:448]: @CathalOConnorRH Should we put the image into quay.io/integreatly?

Comment (briangallagher) [reply test/common/3scale_smtp.go:448]: As discussed, this change will be made in a separate JIRA, when access to quay.io/integreatly/ is received. cc @CathalOConnorRH

**Pull request merged**